src/stylesheets/components/_footer.scss

```diff
@@ -272,3 +272,43 @@ li.usa-footer-primary-content {
     padding-left: 1em;
   }
 }
+
+.usa-social-link {
+  $hit-area: 4.5rem; // used for height and width
+  $left-align-adjust: 1.2rem; // relative left positioning
+  $blank-space: 1rem; // spacing between link hit areas
+
+  background-position: center center;
+  background-repeat: no-repeat;
+  display: inline-block;
+  height: $hit-area;
+  left: $left-align-adjust;
+  margin-left: $blank-space;
+  position: relative;
+  text-align: center;
+  width: $hit-area;
```

Comment:
Hmmmm...either way is fine for me. 

The initial reason - specifically for the hit area one is actually because, according to the CSS coding style, I couldn't put width and height next to each other alphabetically. The other two came about just to further separate the static values from the implementation - OOP-style.

Again, I'm fine for whichever.